chat-widget/src/common/Constants.ts

```diff
@@ -44,6 +44,7 @@ export class Constants {
     public static readonly queuePositionMessageTag = "queueposition";
     public static readonly averageWaitTimeMessageTag = "averagewaittime";
     public static readonly message = "message";
+    public static readonly hiddenTag = "Hidden";
```

Comment:
not relevant, but maybe keep style by not starting with capital letter

Reply:
It's an existing tag being used across services and C1. So, changing it will break the flow.